script/Ciel/Day 7/07_01_CIEL07_1_3B.txt

```diff
@@ -52,7 +53,7 @@ But, Tohno-kun, is your vision that bad?"
 -- 外さないのはただの願掛けです。子供の頃にお世話になった人との約束というか。……まあ、それも」
 // さないのはただの願掛けです is so weird
 I've just sworn not to take them off.
- The long and short of it is that I made a promise with a person who helped me out when I was a kid.
+ The long and short of it is that I made a promise to a person who helped me out when I was a kid.
```

Comment:
I prefer with

Reply:
sure i changed it since promise to is more common but promise with also works for me